scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.12.1] - 2018-03-12
+### Added
+- Added resource, job category, and other caching @scrosby
+- New query functions for API calls of get-(pending|running) jobs by user @scrosby
```

Comment:
I'd also phrase this as optimizations, the way it's written sounds like a new API